### DISCUSSION TITLE
Connect Rasap to Open Network and share it?

### DISCUSSION THREAD

**Discussion by crystalstyl** [RaspAP/raspap-webgui#1534] [Q&A] [answered]
Hello everyone, 

I have successfully configured RaspAP and everything is working so far. 

Now I am facing a problem: 

I want to log on to a "public" WLAN, which does not require a password, but you have to accept it after connecting via Captive. 

Unfortunately, I cannot connect to a public WLAN via the Web GUI? 
In another article it says that the Innsider version supports this function? Is that correct? Or what can I do to make this configuration work? 

Raspberry 4 
integrated Wlan + USB Dongle

**Comment by billz** [comment]
If your public WLAN is using WEP (god forbid) RaspAP won't connect to it. This is clearly indicated in the WiFi client UI:

"WEP access points appear as 'Open'. RaspAP does not currently support connecting to WEP"

**Comment by mike-vivas** [comment]
hi,

same problem here - and yes it is written in the docs and in the UI that WEP networks are not supported - but still: 
the need here would be to join a completely OPEN network, not even protected with WEP.
Many hotels and camping have such open networks and authenticate users via Captive Portal once connected.

is there at least a command line workaround to connect to an OPEN network? or a way to tweak the config?
I am experimenting but I was not successful so far

**Reply by billz** [reply]
Type `/` to search this repo and enter "captive".

**Reply by mike-vivas** [reply]
is this what you mean? https://docs.raspap.com/captive/

unfortunately not the use-case I am interested in

**Reply by billz** [reply]
No, I'm trying to give you some pointers for searching this repo. This topic has been discussed _many_ times.

**Reply by mike-vivas** [reply]
ok thanks for that - but in the past days I was searching here. I looked again and I could not find points directly related to the issue mentioned here.

in particular:
- the captive portal is NOT the issue. I have a working solution for it
- the issue is connecting to a wireless network of type OPEN (no password needed). In the version 3.1.3 this does NOT work from the web interface (menu 'wifi client'). Clicking on 'Connect' does not work, no connection is established.

if I trigger via command line the connection to the very same open network, it works. I can see the captive portal, I can login and I have internet.
Triggering the connection to the same WiFi network from the UI does not work (and therefore no captive portal and no internet)

is it an issue just in 3.1.3? or is it the standard way (that open-networks are not supported)?

**Reply by kalel681** [reply]
I'm currently experiencing the same issue. Attempting to connect to any network without a password results in a failed connection. I've attempted on 2.4g, 5g, and wifi 6 networks, all with the same result.

**Reply by billz** [reply]
It's possible this is a bug. Please be patient while we look into it.

**Reply by kalel681** [reply]
I've created a bug report for this issue. Let me know if you require any further information.

**Reply by billz** [reply, marked as answer]
Relevant issue link #1588. This is now resolved.
